README.md

```diff
@@ -8,6 +8,25 @@ Harvester is an open-source application that downloads data from various public
 [![Test Coverage](https://codeclimate.com/github/slovensko-digital/harvester.ecosystem/badges/coverage.svg)](https://codeclimate.com/github/slovensko-digital/harvester.ecosystem/coverage)
 [![Inch CI](https://inch-ci.org/github/slovensko-digital/harvester.ecosystem.svg)](https://inch-ci.org/github/slovensko-digital/harvester.ecosystem)
 
+## Development  
+
+Harvester is a Ruby-on-Rails application. However, there is no web process present. The application
+runs as a backend worker/scheduler and uses the Rails framework to enforce coding rules for the project. 
+
+For development, Harvester requires:
+
+ - Postgres 9.6+ and `psql` Postgres client `(to store scraped data) 
```

Comment:
Lets mention here that we have prepared docker compose for you.

Reply:
My 2c: the Gemfile (and now docker-compose file) are already pretty good descriptors of external dependencies. Listing them here in README would create a new piece of documentation to keep up to date. The added benefit is rather small -- the set-up we're using is very standard in the Rails world.

My suggestion would be to just point to the docker-compose file and not list the dependencies.

Reply:
Sounds good, will change